test_scripts/API/LastMileNavigation/OnWayPointChange/005_OnWayPointChange_update_internaly_stored_notification_data.lua

```diff
@@ -0,0 +1,123 @@
+-- Item: Use Case 1: Alternative flow 1
+--
+-- Requirement summary:
+-- [OnWayPointChange] As a mobile application I want to be able to be notified on changes 
+-- to Destination or Waypoints based on my subscription
+--
+-- Description:
+-- In case:
+-- 1) One application requested to unsubscribe from receiving notifications on destination & waypoints changes (other mobile applications remain subscribed)
+--    Any change in destination or waypoints is registered on HMI (user set new route, canselled the route, arrived at destination point or crossed a waypoint)
+-- 2) HMI sends the notification about changes to SDL
+
+-- SDL must:
+-- 1) Update of internaly stored notification data in case new notification came from HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonLastMileNavigation = require('test_scripts/API/LastMileNavigation/commonLastMileNavigation')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+
+local firstNotification ={}
+  firstNotification.wayPoints =
+  {{
+      coordinate =
+      {
+        latitudeDegrees = 1.1,
+        longitudeDegrees = 1.1
+      },
+      locationName = "Hotel",
+      addressLines =
+      {
+        "Hotel Bora",
+        "Hotel 5 stars"
+      },
+      locationDescription = "VIP Hotel",
+      phoneNumber = "Phone39300434",
+      locationImage =
+      {
+        value ="icon.png",
+        imageType ="DYNAMIC",
+      },
+      searchAddress =
+      {
+        countryName = "countryName",
+        countryCode = "countryCode",
+        postalCode = "postalCode",
+        administrativeArea = "administrativeArea",
+        subAdministrativeArea = "subAdministrativeArea",
+        locality = "locality",
+        subLocality = "subLocality",
+        thoroughfare = "thoroughfare",
+        subThoroughfare = "subThoroughfare"
+      }
+  } }
+
+  local secondNotification ={}
+  secondNotification.wayPoints =
+  {{
+      coordinate =
+      {
+        latitudeDegrees = 2.1,
+        longitudeDegrees = 2.1
+      },
+      locationName = "Home",
+      addressLines =
+      {
+        "Street 36"
+      },
+      locationDescription = "MyHome",
+      phoneNumber = "009300434",
+      locationImage =
+      {
+        value ="icon.png",
+        imageType ="DYNAMIC",
+      },
+      searchAddress =
+      {
+        countryName = "countryName",
+        countryCode = "countryCode",
+        postalCode = "postalCode",
+        administrativeArea = "administrativeArea",
+        subAdministrativeArea = "subAdministrativeArea",
+        locality = "locality",
+        subLocality = "subLocality",
+        thoroughfare = "thoroughfare",
+        subThoroughfare = "subThoroughfare"
+      }
+  } }
+
+
+--[[ Local Functions ]]
+local function FirstOnWayPointChangeAndSubscribeApp2(self)
```

Comment:
@LitvinenkoIra It's better to split this function into 2 parts: OmWayPointChange and Subscribe